### PR TITLE
Rfolco/all in one json

### PR DESCRIPTION
**Pull request by rafaelfolco**

### CODE REVIEW

File: JSON/schema.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "mv-params": {
+            "type": "object"
+        },
+        "passthru-args": {
+            "type": "string"
+        }
+    },
+    "required": [
+        "mv-params"
+    ]
```
Comment (k-rister) [inline JSON/schema.json:24]: I think we probably want an `"additionalProperties": false` here, right?

**Review by k-rister — CHANGES_REQUESTED** [review]

**Comment by k-rister** [comment]
I'm also wondering if we should version this schema, this is likely to eventually be our most user facing interface and it seems like it would be a good idea to version it for if/when changes are made.

File: JSON/schema.json (hunk 1 above)
Comment (rafaelfolco) [reply JSON/schema.json:24]: Agreed, the fact that the tool only allows specific values for --config <config> via code may not be enough

**Comment by rafaelfolco** [comment]
> I'm also wondering if we should version this schema, this is likely to eventually be our most user facing interface and it seems like it would be a good idea to version it for if/when changes are made.

ack

**Comment by rafaelfolco** [comment]
> Not sure I understand the use of the additionalProperties at the second level...can you explain your thinking?
for tags, endpoint and passthru-args, we may want to define more rules on what key:value pairs are accepted. So I am leaving as additionalProperties: True, so we don't restrict anything now. I know this is not making any difference at the moment, but as we make gradual changes to the inner levels, we may need this to make sure we are not blocking possible key:value pairs. As I write this long sentence I realize that we don't need it now, so I am ok to delete for this PR. End of story.

**Pull request merged**